config/initializers/warden_github.rb

```diff
@@ -2,8 +2,8 @@
 
 module WardenGithubStrategyExtensions
   def authenticate!
-    if scope == :sidekiq && session[:sidekiq_user].present?
-      success!(session[:sidekiq_user])
+    if (scope == :sidekiq && session[:sidekiq_user].present?) ||
+       (scope == :coverband && session[:coverband_user].present?)
```

Comment:
Do you need to put this success!(session[:sidekiq_user]) line back?

Reply:
@LindseySaari Thank you. I had to remove a lot for dev testing. Thank you!